framework_tvb/tvb/core/adapters/abcuploader.py

```diff
@@ -69,6 +84,9 @@ class ABCUploader(ABCSynchronous, metaclass=ABCMeta):
     """
     LOGGER = get_logger(__name__)
 
+    def __init__(self):
+        super(ABCUploader, self).__init__()
```

Comment:
If nothing is left in a method other than calling super, does it make sense to have it ?

Reply:
No.